pkg/lib/cockpit/_internal/transport.d.ts

```diff
@@ -0,0 +1,28 @@
+import { JsonObject } from './common';
+
+interface ChannelControlMessage extends JsonObject {
+    command: string;
+    channel: string;
+}
+export type ControlHandler = (message: ChannelControlMessage) => void;
+
+// We might have either `string` or `Uint8Array` appearing here, and we want to
+// be able to treat things as either from the Channel code.  We trust
+// cockpit-ws to send the correct frame type.  We might clean this up later.
+export type PayloadHandler = (message: any) => void; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Out of interest, why can't this be `string | Uint8Array`? Like `send_message`? Also, could this be factored into a `type Message`?

Reply:
Thanks for the explanation!